index.js

```diff
@@ -33,7 +33,9 @@ module.exports = function toJSON(schema) {
 
       //Set ID
       if (ret._id && typeof ret._id === 'object' && ret._id.toString) {
-        ret.id = ret._id.toString();
+        if (ret.id == null) {
```

Comment:
Actually I've think about this, but I'm still wondering should "id" nullable ?
But I think will change it to what you recommend 👍